### PR TITLE
Add LXC deploy script

### PR DESCRIPTION
**Pull request by jungziege**
This adds a script for deploying LXC images to our infrastructure.

### CODE REVIEW

File: scripts/release-LXC-container [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+SUPPORTED_REGIONS=(
+    ap-northeast-1
+    ap-northeast-2
+    ap-southeast-1
+    ap-southeast-2
+    eu-central-1
+    eu-west-1
+    sa-east-1
+    us-east-1
```
Comment (gordonsyme) [inline scripts/release-LXC-container]: ~ There's a `us-east-2` as well now.

File: scripts/release-LXC-container [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    us-west-1
+    us-west-2
+)
+
+function publish_container {
+    REGION=$1
+    S3_URL="s3://circleci-enterprise-assets-$REGION/containers/$NAME.tar.gz"
+    aws --region $REGION s3 cp --acl public-read $FILE $S3_URL
```
Comment (gordonsyme) [inline scripts/release-LXC-container]: Relying on the `aws` binary present on the instance can lead to hard to debug errors unless you can be certain it's a recent enough version to support whatever the script is doing. I wouldn't be surprised if the currently installed `aws` chokes on `us-east-2` for instance.

File: scripts/release-LXC-container (hunk 1 above)
Comment (jungziege) [inline scripts/release-LXC-container]: It's a good suggestion, but there are a couple of other places that we would need to update to support `us-east-2`. I don't think there is much cost associated with it either, but also don't want to track down all those places until someone asks for us-east-2 support.

File: scripts/release-LXC-container (hunk 2 above)
Comment (jungziege) [inline scripts/release-LXC-container]: We already rely on / ship the aws cli as part of the CI/CD process for this repo. Do you think it's worth adding it to the dependencies section?

Comment (gordonsyme) [inline scripts/release-LXC-container]: If it's pulling the most recent aws-cli for when the image is built anyway there won't be any issues.

**Pull request merged**